user/urls.py

```diff
@@ -1,8 +1,12 @@
-from django.urls import path
+from django.urls import path, include
 
 from . import views
 
 urlpatterns = [
     path('', views.TopView.as_view(), name='top'),
     path('signup', views.SignupView.as_view(), name='signup'),
+    path('', include('django.contrib.auth.urls')),
+    path('home', views.HomeView.as_view(), name='home'),
+    path('login',include('django.contrib.auth.urls')),
+    path('logout',include('django.contrib.auth.urls')),
```

Comment:
```suggestion
    path('login',include('django.contrib.auth.urls')),
    path('logout',include('django.contrib.auth.urls')),
```

これって必要でしょうか？🤔
8行目で定義されている気がします